### PR TITLE
Feat/net097 70/update profile

### PR DESCRIPTION
**Pull request by 5haki6**

### CODE REVIEW

File: MilitaryManager.Units.API/appsettings.Development.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=BAGUTA;Initial Catalog=Units;User ID=sa;Password=password1;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    "DefaultConnection": "Data Source=localhost,1433;Initial Catalog=Units;User ID=User_2;Password=qwe123qwe;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```
Comment (Zakerman110) [inline MilitaryManager.Units.API/appsettings.Development.json]: Please, don't use user and password for local connection, it's just enough for Windows Authentication (integrated security)

File: MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts [hunk 2]
```diff
@@ -17,7 +18,10 @@ export class ProfileComponent implements OnInit {
   position: string | null | undefined = '';
   unitModel: UnitModel;
 
-  constructor(private _unitUserService : UnitUserService, private _authService: AuthService, private _unitService: UnitsService) {
+  constructor(private _unitUserService : UnitUserService,
+              private _authService: AuthService,
+              private _unitService: UnitsService,
```
Comment (Zakerman110) [inline MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts:23]: No need to inject every service apart, you can inject only ApiService that includes all other services

File: docker-compose.override.yml [hunk 3]
```diff
@@ -29,7 +29,7 @@ services:
       - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=https://+:443;http://+:80
       - identityUrl=http://militarymanager.identity.server
-      - ConnectionStrings__DefaultConnection=Server=host.docker.internal;Database=Units;User ID=sa;Password=password1;MultipleActiveResultSets=true;Application Name=MilitaryManager.Units.API
+      - ConnectionStrings__DefaultConnection=Server=host.docker.internal;Database=Units;User ID=User_2;Password=qwe123qwe;MultipleActiveResultSets=true;Application Name=MilitaryManager.Units.API
```
Comment (Zakerman110) [inline docker-compose.override.yml]: Don't change docker-compose

File: docker-compose.override.yml [hunk 4]
```diff
@@ -41,7 +41,7 @@ services:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=https://+:443;http://+:80
-      - ConnectionStrings__DefaultConnection=Server=host.docker.internal;Database=IdentityServer_AspNet_Identity;User ID=Gwa_System;Password=T&t137S5tP!+3SMJQ<r0KJ;MultipleActiveResultSets=true;Application Name=IdentityServer
+      - ConnectionStrings__DefaultConnection=Server=host.docker.internal;Database=IdentityServer_AspNet_Identity;User ID=User_2;Password=qwe123qwe;MultipleActiveResultSets=true;Application Name=IdentityServer
```
Comment (Zakerman110) [inline docker-compose.override.yml]: Don't change docker-compose

**Review by Alex-knu — APPROVED** [review]

File: MilitaryManager/client-app/src/app/components/profile/update-profile/update-profile.component.ts [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+  pos: PositionModel[] = [];
+  attributes: AttributeModel[] = [];
+  ranks: RankModel[] = [];
+  uniforms: string[] = [];
+  blood_types: string[] = [];
+
+  unitModel: UnitModel;
+    readonly weightValueValidator = (control: FormControl) => {
+
+      if(control.value < 40 || control.value > 150) {
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/components/profile/update-profile/update-profile.component.ts]: Move these validator methods to a separate class.

**Review by vladtymo — CHANGES_REQUESTED** [review]

**Review by vladtymo — CHANGES_REQUESTED** [review]
Add an opportunity to create and remove custom attributes.

**Review by vladtymo — APPROVED** [review]

**Pull request merged**